clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -97,7 +94,16 @@
       "\t$0",
       "END-IF."
     ],
-    "description": "IF variable-1 IS ALPHABETIC"
+    "description": "A COBOL \"class test\" used to determine whether the contents of a variable contains specific upper/lower case values. Used in conjunction with CMPR2/NOCMPR2 as compilation directives. https://www.ibm.com/docs/en/cobol-zos/6.3?topic=expressions-class-condition"
+  },
+  "ALPHANUMERIC": {
+    "prefix": "ALPHANUMERIC",
+    "body": [
+      "IF ${1:variable-1} IS ${2|ALPHANUMERIC,ALPHANUMERIC-EDITED|} THEN",
+      "\t$0",
+      "END-IF."
+    ],
+    "description": "A COBOL \"class test\" which is used to determine whether the contents of a variable contains only letters and numbers (no special character) values. https://www.ibm.com/docs/en/search/alphanumeric?scope=SS6SG3_6.3.0"
```

Comment:
A COBOL \"class test\" which is used to determine whether the contents of a variable contain only letters and numbers (no special characters).